services/api/src/clients/harborClient.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+const got = require('got');
+const R = require('ramda');
+
+const { HARBOR_ADMIN_PASSWORD } = process.env;
+
+const lagoonHarborRoute = R.compose(
+  R.defaultTo('http://172.17.0.1:8084'),
```

Comment:
can't the default be just `harbor-nginx:8080`? We want to run as much within the docker-compose networking instead of having the api connect to an "external" port.

Reply:
Because the docker-compose harbor is running in a different docker network than the k3d docker network, we have to use this forwarded port to allow access between the two networks.